packages/app/src/cli/services/deploy/bundle.ts

```diff
@@ -57,23 +64,31 @@ export async function bundleAndBuildExtensions(options: BundleOptions) {
             },
           },
         })),
-        ...options.app.extensions.function.map((functionExtension) => {
```

Comment:
Could we keep this here and instead do the check `if (!bundleTheme && !bundleUI) return` at the end? So something like

```
if (!bundleTheme && !bundleUI) {
  return
} else {
  const bundlePath = joinPath(options.bundleRootPath, `bundle.zip`)
    await zip({
      inputDirectory: bundleDirectory,
      outputZipPath: bundlePath,
    })
    return bundlePath
}

Reply:
done!